app/mailers/conversation_reply_mailer.rb

```diff
@@ -80,7 +80,8 @@ def last_outgoing_message
   end
 
   def assignee_name
-    @assignee_name ||= @agent&.available_name || 'Notifications'
+    sender_name = @message&.sender&.name || @agent&.available_name
+    @assignee_name ||= sender_name || 'Notifications'
   end
```

Comment:
Should we change the function to `sender_name` instead?

Reply:
Sure, Adding this as a new commit. It will have more impact.

Reply:
@sojan-official I have updated the method's name to sender_name